sql/item_mods.sql

```diff
@@ -23062,7 +23062,7 @@ INSERT INTO `item_mods` VALUES (14645,5,30);   -- MP: 30
 INSERT INTO `item_mods` VALUES (14645,21,-30); -- LIGHT_RES: -30
 
 -- Shadow Ring
-INSERT INTO `item_mods` VALUES (14646,255,5);  -- DEATHRES: 5
+INSERT INTO `item_mods` VALUES (14646,255,20);  -- DEATHRES: 20
```

Comment:
Please correct the alignment of the comment.

Reply:
Done